<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDModifierUnGroupeClient</class>
 <widget class="QDialog" name="BDModifierUnGroupeClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Groupes en cours</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidgetGroupesEnCours">
        <attribute name="headerDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>IdGroupe</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Libellé</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Responsable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nb pers enreg</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_terminer">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Terminer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxMembres">
     <property name="title">
      <string>Membres du groupe (nb membres)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidgetMembres">
        <attribute name="headerDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>IdClient</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Client</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Logé</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_ajouter">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_retirer">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Retirer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="formAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_choisir">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Choisir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_fermer">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
